.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
```

Comment:
You can avoid having to add the `/* globals describe it expect */` comments at the top of the test files by adding `"jest": true` in here:
```
    "browser": true,
    "jest": true,
    "es2021": true
```